lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AppProxyController < ApplicationController
+  if Rails.env.development? || Rails.env.test?
+   around_action :shopify_session
+  else
+   include ShopifyApp::AppProxyVerification
+  end
```

Comment:
>I thought dev shop was like any other shop except it didnt have a validity nor billing fn. attached to it.

It is. You can't *run* Shopify on your own computer. But you can use a tunnelling service - we use [forward](https://forwardhq.com/) at the moment so we can use app proxies, webhooks, etc. in development.